app/controllers/games_controller.rb

```diff
@@ -67,7 +72,7 @@ def destroy
   end
   
   def add_player
-    current_player.join_game!(@game)
+    current_player.join_game!(@game, current_player)
```

Comment:
This is throwing an error for me because the join_game! method is expecting only one argument.